gcc/testsuite/rust/compile/builtin_macro_eager3.rs

```diff
@@ -0,0 +1,21 @@
+#[rustc_builtin_macro]
+macro_rules! include_str {
+  () => {{}};
+}
+
+macro_rules! foo {
+    () => {
+        "foo.txt"
+    };
+}
+
+macro_rules! foo {
+    () => {
+        "bar.txt"
+    };
+}
+
+fn main () {
+  include_str!(foo!()); // ok
+  include_str!(foo!(), bar!()); // ok
```

Comment:
This is not a proper invocation of the macro sadly :) error from `rustc`:

```rust
error: include_str! takes 1 argument
  --> test.rs:22:5
   |
22 |     include_str!(foo!(), bar!()); // ok
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `include_str` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Also, I can't find any file named `bar.txt` or `foo.txt` in the repo, so I think you might have forgotten to push them?